s/features/notes/components/notes/xiome-notes.css.ts

```diff
@@ -1,18 +1,100 @@
-
-import {css} from "../../../../framework/component.js"
+import { css } from "../../../../framework/component.js";
 export default css`
-
 :host {
 	display: block;
-}
+	}
 
 [data-tab] {
 	font-weight: normal;
+	margin-left: 20px;
+	border: none;
+	border-top: 0px;
+	border-left: 0px;
+	border-right: 0px;
+	border-style: solid;
+	outline: none;
 }
 
-[data-tab][data-active='true'] {
+[data-tab][data-active="true"] {
 	font-weight: bold;
 	color: bisque;
 }
 
-`
+.tabs {
+	display: flex;
+	justify-content: center;
+}
+
+.tab-new,
+.tab-old {
+}
+
+.note-container {
+	border-style: solid;
+	padding: 20px;
+	width: 30%;
+}
+.note-body {
+	border-style: solid;
+	border-width: 3px 3px;
+	width: auto;
+	color: gray;
+	list-style-type: none;
+	padding: 10px 10px 10px 10px;
```

Comment:
in modern css, we want to avoid using `px` units, preferring instead to use flexible `em` units wherever possible